Sender: LSF System <lsfadmin@e2105>
Subject: Job 32074364: <python SVM2.py 250 278> in cluster <euler> Exited

Job <python SVM2.py 250 278> was submitted from host <euler07> by user <lionelt> in cluster <euler>.
Job was executed on host(s) <e2105>, in queue <normal.24h>, as user <lionelt> in cluster <euler>.
</cluster/home/lionelt> was used as the home directory.
</cluster/home/lionelt/machineLearning/MLP2/preproc> was used as the working directory.
Started at Wed Nov 16 13:30:07 2016
Results reported on Wed Nov 16 13:30:08 2016

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
python SVM2.py 250 278
------------------------------------------------------------

Exited with exit code 2.

Resource usage summary:

    CPU time :               0.03 sec.
    Total Requested Memory : 8000.00 MB
    Delta Memory :           -
    (Delta: the difference between Total Requested Memory and Max Memory.)

The output (if any) follows:

python: can't open file 'SVM2.py': [Errno 2] No such file or directory
Sender: LSF System <lsfadmin@e3034>
Subject: Job 32074365: <python plot_peaks.py 250 278> in cluster <euler> Exited

Job <python plot_peaks.py 250 278> was submitted from host <euler07> by user <lionelt> in cluster <euler>.
Job was executed on host(s) <e3034>, in queue <normal.24h>, as user <lionelt> in cluster <euler>.
</cluster/home/lionelt> was used as the home directory.
</cluster/home/lionelt/machineLearning/MLP2/preproc> was used as the working directory.
Started at Wed Nov 16 13:31:14 2016
Results reported on Wed Nov 16 13:31:17 2016

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
python plot_peaks.py 250 278
------------------------------------------------------------

Exited with exit code 2.

Resource usage summary:

    CPU time :               0.04 sec.
    Total Requested Memory : 8000.00 MB
    Delta Memory :           -
    (Delta: the difference between Total Requested Memory and Max Memory.)

The output (if any) follows:

python: can't open file 'plot_peaks.py': [Errno 2] No such file or directory
Sender: LSF System <lsfadmin@e1379>
Subject: Job 32074428: <python plot_peaks.py 250 278> in cluster <euler> Exited

Job <python plot_peaks.py 250 278> was submitted from host <euler07> by user <lionelt> in cluster <euler>.
Job was executed on host(s) <e1379>, in queue <normal.24h>, as user <lionelt> in cluster <euler>.
</cluster/home/lionelt> was used as the home directory.
</cluster/home/lionelt/machineLearning/MLP2/preproc> was used as the working directory.
Started at Wed Nov 16 13:35:43 2016
Results reported on Wed Nov 16 13:35:44 2016

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
python plot_peaks.py 250 278
------------------------------------------------------------

Exited with exit code 2.

Resource usage summary:

    CPU time :               0.05 sec.
    Total Requested Memory : 8000.00 MB
    Delta Memory :           -
    (Delta: the difference between Total Requested Memory and Max Memory.)

The output (if any) follows:

python: can't open file 'plot_peaks.py': [Errno 2] No such file or directory
